### ISSUE TITLE
V 4.0 not working on Nanopool XMR

### ISSUE THREAD

**Issue by michaeldlc** [nanopool/Claymore-XMR-CPU-Miner#11]
I'm receiving this error on nanopool XMR. 

cannot find block data:
{"id":4,"jsonrpc":"2.0","result":null,"error":{"code":-1,"message":"Lowdifficultyshare"}}
cannot convert blob data

**Comment by dateno1** [comment]
me too

i found same message on new version testing

**Comment by michaeldlc** [comment]
Received a reply from nanopool support. Use "-pow7 1" at the end of your config.  Works for me.

**Comment by Joz76** [comment]
me too. I use claymore CPU mining in 4 machines, 2 win7 and 2 win10. 2 win7 make the same error, the 2 win10 running without issues, but the pool (nanpool) shows all of 4 machines hash rate is zero.

**Comment by Joz76** [comment]
Yes, the "-pow7 1" works for me too, and have a new Claymore CPU miner 4.0 with Dev fee 0% :-)

**Comment by samapal** [comment]
adding -pow7 1 but not works for me
with hash rate > 100 hash per second there is no any found shares on all day mining 24 hours

**Comment by Joz76** [comment]
Samapal: "-pow7 1" needed Claymore CPU miner 4.0, not include in 3.9. I minig with my noteook CPU (i7-5600U), that about 90-100 h/s, and it works, on average 4 share / hour
